cli/templates/.env

```diff
@@ -0,0 +1,48 @@
+# Learn more about the Grouparoo environment variables @ https://www.grouparoo.com/docs/support/environment
```

Comment:
Why was this needed?  That's surprising

Reply:
That stack overflow solution for case sensitive commits was the culprit for files going missing on this PR and the last!  Definitely going to watch the commit changes more closely if I use it again.